readme.didact.md

```diff
@@ -178,10 +178,10 @@ To run the integration, you need to link it to the proper configuration, that de
 
 ### 4.1 [Alternative 1] Using the test Minio server
 
-As alternative, to connect the integration to the **test Minio server** deployed before using the [test/MinioCustomizer.java](didact://?commandId=vscode.open&projectFilePath=test/MinioCustomizer.java "Opens the customizer file"){.didact} class:
+As alternative, to connect the integration to the **test Minio server** deployed before:
 
 ```
-kamel run --name api test/MinioCustomizer.java API.java --property-file test/minio.properties --open-api openapi.yaml -d camel-openapi-java
+kamel run --name api API.java --property-file test/minio.properties --open-api openapi.yaml -d camel-openapi-java
 ```
 ([^ execute](didact://?commandId=vscode.didact.sendNamedTerminalAString&text=camelTerm$$kamel%20run%20--name%20api%20test/MinioCustomizer.java%20API.java%20--property-file%20test/minio.properties%20--open-api%20openapi.yaml%20-d%20camel-openapi-java&completion=Integration%20run. "Opens a new terminal and sends the command above"){.didact})
```

Comment:
Remember to change the didact script as well. Also, some options (like the openapi ones) can be embedded as modeline

Reply:
done